src/extensions/surveys/surveys-utils.tsx

```diff
@@ -534,7 +538,11 @@ export const sendSurveyEvent = (
     survey: Survey,
     posthog?: PostHog
 ) => {
-    if (!posthog) return
+    // TODO: does it need readOnly like dismissedSurveyEvent?
```

Comment:
@lucasheriques since you are working on the preview bits, do we need the readOnly here too? is the preview sending dismissed events?

Reply:
no the preview doesn't send and kind of events

Reply:
@lucasheriques so should we remove the checks from the other methods?